lib/metadata/oneHotMetadata.hpp

```diff
@@ -0,0 +1,47 @@
+#ifndef ONEHOT_METADATA
+#define ONEHOT_METADATA
+
+#include "Telescope.hpp"
+#include "buffer.h"
+#include "chimeMetadata.hpp"
+#include "datasetManager.hpp"
+#include "metadata.h"
+
+#include <sys/time.h>
+#include <vector>
+
+#pragma pack()
+
+struct oneHotMetadata {
+    struct chimeMetadata chime;
+    int frame_counter;
+    std::vector<int> hotIndices;
```

Comment:
We should change this to a fixed size array, or if we plan to do that in the next PR, at least put a big TODO here to not use vector.